xendit-java-lib/src/main/java/com/xendit/model/DirectDebitPayment.java

```diff
@@ -0,0 +1,261 @@
+package com.xendit.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.xendit.Xendit;
+import com.xendit.exception.XenditException;
+import com.xendit.model.LinkedAccountEnum.ChannelCode;
+import com.xendit.network.RequestResource;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
+public class DirectDebitPayment {
+  @SerializedName("id")
+  private String id;
+
+  @SerializedName("reference_id")
+  private String referenceId;
+
+  @SerializedName("channel_code")
+  private ChannelCode channelCode;
+
+  @SerializedName("payment_method_id")
+  private String paymentMethodId;
+
+  @SerializedName("currency")
+  private String currency;
+
+  @SerializedName("amount")
+  private Number amount;
```

Comment:
Please make sure this accepts decimals, same with refunded_amount

Reply:
Yes `Number` accepts float/double as well. But for IDR it doesn't accept decimals though? And somehow I can't test it for PHP because I can't get the ID with `la` prefix because the status will always be PENDING. Am I missing something?

Reply:
Got it on `Number`.

For pending linked accounts: When linking for PH channels, you would need to access a web page through a url returned in `/linked_account_tokens/auth` (authorizer_url) and login using the mock accounts to complete the linking. 
reference: https://docs.xendit.co/direct-debit/testing-direct-debit/account-linking#bank-account

Reply:
Ah I see.. But are these credentials correct?
![image](https://user-images.githubusercontent.com/60346343/126730449-476bedc6-ca6b-4c48-87cc-b64085b3f527.png)
Because I'm redirected to failure URL after putting in those credentials

Reply:
Issue fixed, yes it accepts decimal but it must be in PHP